hack/verify-actuator-pkg.sh

```diff
@@ -2,5 +2,6 @@
 
 set -e
 
-go get -mod=readonly -u github.com/openshift/cluster-api-actuator-pkg
+# go get -mod=readonly -u github.com/openshift/cluster-api-actuator-pkg
+go get -u github.com/openshift/cluster-api-actuator-pkg
```

Comment:
Why did we drop the mod readonly flag here? Also, should remove the comment if we don't need this? 

Also, didn't you remove the verify-actuator-pkg target from the makefile? Do we still need this?

Reply:
The -mod=readonly flag does not exist for `go get`.
Yes the target is not in the makefile anymore. So if we dont need the script for anything else it could be removed as well.